### PR TITLE
Add config testing to device-mqtt

### PR DESCRIPTION
**Pull request by farshidtz**

### CODE REVIEW

**Review by MonicaisHer — APPROVED** [review]

File: test/suites/device-mqtt/config_test.go [hunk 1]
```diff
@@ -3,42 +3,31 @@ package test
...
-	t.Run("change-the-maximum-startup-duration", func(t *testing.T) {
-		startupDurationValue := "120"
+		const newPort = "56789"
 
-		utils.Exec(t, "sudo snap set edgex-device-mqtt startup-duration="+startupDurationValue)
-
-		stdout, _ := utils.Exec(t, "sudo snap get edgex-device-mqtt startup-duration")
-		assert.Equal(t, startupDurationValue+"\n", stdout, "maximum startup-duration does not set successfully")
+		// make sure the port is available before using it
+		utils.CheckPortAvailable(t, newPort)
```
Comment (MonicaisHer) [inline test/suites/device-mqtt/config_test.go:23]: How do you distinguish if the `newPort` will be used by device-mqtt or other services?
Do we need to first make sure `newPort`  is not been used by any service?

Comment (farshidtz) [reply test/suites/device-mqtt/config_test.go:23]: I think that's exactly what I'm doing on line 23, i.e. checking if the port is available (free) before trying to let the mqtt service use it. If port is used by another server, line 23 will throw an error and tests would stop.

Comment (MonicaisHer) [reply test/suites/device-mqtt/config_test.go:23]: `sudo lsof -nPi :%s | true` does't throw me error neither using an used or unused port. What kind of error you got?

Comment (farshidtz) [reply test/suites/device-mqtt/config_test.go:23]: Good catch. A pipe is missing. It should be `||` (see https://github.com/canonical/edgex-snap-testing/pull/17) In the function implementation, if stdout is empty, it means the port is available.

**Pull request merged**